addon/components/validated-input/render.hbs

```diff
@@ -86,6 +86,24 @@
         @update={{@update}}
         ...attributes
       />
+    {{else if (and
+      (eq @type "date") (not-eq this.dateComponent this.inputComponent)
```

Comment:
Why `(not-eq this.dateComponent this.inputComponent)`? Let's just render the `this.dateComponent` whether it's custom or the default input component, no?

Reply:
Because the `this.dateComponent` recieves all attributes as arguments since we know that this will be an external component i would prefer to not pass html attributes here already and let the component handle the html attributes:
```hbs
@autocomplete={{@autocomplete}}
@autofocus={{@autofocus}}
@disabled={{@disabled}}
@id={{@inputId}}
@name={{or @inputName @name}}
@placeholder={{@placeholder}}
@type={{@type}}
@value={{@value}}
@isInvalid={{@isInvalid}}
@isValid={{@isValid}}
@setDirty={{@setDirty}}
@update={{@update}}
...attributes
```

Reply:
And this wouldn't work for a HTML5 date input.